src/main/kotlin/org/radarbase/push/integration/garmin/backfill/route/GarminRoute.kt

```diff
@@ -28,13 +27,14 @@ abstract class GarminRoute(
                 userRepository.getUserAccessTokenSecret(user)
             )
         )
-
+        val parameters = oauth1.getParams()
+        val signature = userRepository.getOAuthSignature(user, url,"GET", parameters)
         val request = Request.Builder()
```

Comment:
you can init the params here and then create the `Oauth1Signing (oauthKeys, params)`. The params will contain the signature too.

Reply:
I also removed `oAuthKeys` since we're inserting the token in the params already. Is that fine?